src/main/java/co/rsk/federate/signing/hsm/advanceblockchain/HSMBookkeepingService.java

```diff
@@ -5,6 +5,7 @@
 import co.rsk.federate.signing.hsm.client.HSMBookkeepingClient;
 import co.rsk.federate.signing.hsm.message.AdvanceBlockchainMessage;
 import co.rsk.net.NodeBlockProcessor;
+import org.bitcoinj.store.BlockStoreException;
```

Comment:
Not sure this is the right exception to throw here. This is a bitcoinj exception, here we are dealing with RSK blocks and RSK storage

Reply:
Yes, Not The Right Exception. Updated